update rtl.css

islam2hamy opened the pull request:
Signed-off-by: islam info@smfarabic.com

delete , edit , and adding some lines to make it work with the smf2.1 , During using RTL language .

islam2hamy committed:
update rtl.css

Signed-off-by: islam <info@smfarabic.com>

XinYenFon commented on Themes/default/css/rtl.css:
```diff
@@ -14,68 +14,73 @@
 td.lastpost {
 	border-left: 1px solid #ddd;
 	border-right: 0px !important;
 }
```
we don't have td.lastpost anymore. I don't think we need this anymore.

islam2hamy replied:
i will delete it

XinYenFon commented on Themes/default/css/rtl.css:
```diff
@@ -610,6 +528,20 @@ div#admin_menu {
+/* h3 HD Icons */
+h3.search_hd {
+    padding: 8px 45px 6px;
+	background-position : 99% 50%;
```
why bg-position here?

islam2hamy replied:
to show the icon from right , not left .

XinYenFon replied:
why don't you just use float: right; ?

islam2hamy replied:
it will not work , i have tried a lot of solutions, "bg-position" was the best solution
if you have another solutions , let me know

XinYenFon commented on Themes/default/css/rtl.css:
```diff
@@ -610,6 +528,20 @@ div#admin_menu {
+}
+h3.profile_hd {
+	padding: 8px 50px 8px 0;
+	background-position : 99% 50%;
```
and also here?

XinYenFon commented on Themes/default/css/rtl.css:
```diff
@@ -677,21 +609,30 @@ div#admin_menu {
+	min-width: 95%;
+	float: right;
+}
+/* Introduce New Title Bar -Islam */
```
as much as we appreciate your help please do not leave your mark.

islam2hamy replied:
sorry forgot to delete it , i use it as guide for me

islam2hamy committed:
update rtl.css

Signed-off-by: islam <info@smfarabic.com>

XinYenFon commented on Themes/default/css/rtl.css:
```diff
@@ -102,104 +101,101 @@ dl.settings img {
+	float: left;
+}
+
+
```
extra bump here :P

XinYenFon commented on Themes/default/css/rtl.css:
```diff
@@ -677,21 +603,30 @@ div#admin_menu {
+/* Introduce New Title Bar */
+
+.title_bar {
+
```
extra two bump here :P

XinYenFon commented:
I pointed some small issues to you, if you fix them I approve this PR.

islam2hamy committed:
Update rtl.css

islam2hamy committed:
Update rtl.css

islam2hamy commented:
Thank you Antes ,
still only one . "Line : 538"

XinYenFon commented:
If its not a problem, its not a problem :) Thanks for fast responses, this PR good to go. :( But you forgot to sign-off your last two commits.

islam2hamy committed:
Update rtl.css

Signed-off-by: islam <info@smfarabic.com>

islam2hamy commented:
what about now ? :)

islam2hamy committed:
Update rtl.css

update with some Antes and live627 changes
because my file woth old,

285968d , 8cc79a6 , f13f3fa , 88088d6

Signed-off-by: islam<info@smfarabic.com>

live627 commented:
You should use tabs for indenting, not spaces.

islam2hamy commented:
How to do that ?

live627 commented:
nvm
